doc/nrf/config_and_build/bootloaders_and_dfu/bootloader_adding.rst

```diff
@@ -286,35 +297,39 @@ The following section describes how to add it to your secure bootloader chain.
 Adding MCUboot as an upgradable bootloader
 ==========================================
 
-To use MCUboot as an upgradable bootloader, the application must already use the |NSIB| as the immutable bootloader.
-MCUboot is added to its boot chain by including the :kconfig:option:`CONFIG_BOOTLOADER_MCUBOOT` Kconfig option with either the build command or in the application's :file:`prj.conf` file:
+To use MCUboot as an upgradable bootloader to your application, complete the following steps:
 
-.. code-block::
+1. :ref:`Add |NSIB| as the immutable bootloader <ug_bootloader_adding_immutable_b0>`.
+#. Add MCUboot to the boot chain by including the :kconfig:option:`CONFIG_BOOTLOADER_MCUBOOT` Kconfig option with either the build command or in the application's :file:`prj.conf` file:
 
-   west build -b nrf52840dk_nrf52840 zephyr/samples/hello_world -- \
-   -DCONFIG_SECURE_BOOT=y \
-   -DCONFIG_BOOTLOADER_MCUBOOT=y
+   .. code-block::
 
-|how_to_configure|
+      west build -b nrf52840dk_nrf52840 zephyr/samples/hello_world -- \
+      -DCONFIG_SECURE_BOOT=y \
+      -DCONFIG_BOOTLOADER_MCUBOOT=y
 
-MCUboot can use the cryptographic functionality exposed by the immutable bootloader, reducing the flash memory usage for MCUboot to less than 16 kB.
-To enable this configuration, apply both the :file:`prj_minimal.conf` Kconfig project file and the :file:`external_crypto.conf` Kconfig fragment for the MCUboot image:
+   |how_to_configure|
 
-.. code-block::
+#. Optionally, you can configure MCUboot to use the cryptographic functionality exposed by the immutable bootloader and reduce the flash memory usage for MCUboot to less than 16 kB.
+   To enable this configuration, apply both the :file:`prj_minimal.conf` Kconfig project file and the :file:`external_crypto.conf` Kconfig fragment for the MCUboot image:
 
-   west build -b nrf52840dk_nrf52840 zephyr/samples/hello_world -- \
-   -DCONFIG_BOOTLOADER_MCUBOOT=y \
-   -DCONFIG_SECURE_BOOT=y \
-   -Dmcuboot_CONF_FILE=prj_minimal.conf \
-   -Dmcuboot_OVERLAY_CONFIG=external_crypto.conf
+   .. code-block::
+
+      west build -b nrf52840dk_nrf52840 zephyr/samples/hello_world -- \
```

Comment:
Testing this command with v2.4.0, I get the error
```
CMake Error at C:/ncs/v2.4.0/nrf/modules/mcuboot/CMakeLists.txt:292 (file):
  file STRINGS file "C:/ncs/v2.4.0/nrf/modules/mcuboot/prj_minimal.conf"
  cannot be read.
```
I also got a colleguea to test, and they say the same

Reply:
I haven't touched this command, just changed formatting style around it. Can you open a documentation bug for the developer who added it? It's a "Won't Fix" for this PR's scope for me.

Reply:
Created https://nordicsemi.atlassian.net/browse/NCSIDB-1056